### PR TITLE
Wrap figcaption to make Firefox read img alt first

### LINKED ISSUE #6006: Firefox: Alt text for images on article pages not being declared before the caption beneath the image
**What**

The Alt text for the images within articles is being read out after the caption beneath the image which makes the page compositon confusing if you are using a screen reader. 

It is  not in the correct order. The screen reader should be reading vertically down the page so the image should be declared with the alt text and then the quote under it. 

This may be because there is what looks like an image/fig caption being used for a quote. If there is a fig caption then the alt text would usually be excluded as it would be read twice. So its possible it is due to composition rather than code.

**Where**

Screenshot 2021-01-21 at 16 16 56

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ 

Fixes #6006

### CODE REVIEW

File: common/views/pages/_document.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {
   GoogleTagManagerNoScript,
   GaDimensions,
 } from '../../services/app/google-analytics';
+import { getCookie } from 'cookies-next';
```
Comment (davidpmccormick) [inline common/views/pages/_document.tsx:20]: not sure how I got this in here – not to do with this pr

File: common/views/pages/_document.tsx [hunk 2]
```diff
@@ -39,11 +40,13 @@ function renderSegmentSnippet() {
 type DocumentInitialPropsWithTogglesAndGa = DocumentInitialProps & {
   toggles: Toggles;
   gaDimensions?: GaDimensions;
+  gaUserId?: string;
```
Comment (davidpmccormick) [inline common/views/pages/_document.tsx:43]: or this

File: common/views/components/Caption/Caption.tsx [hunk 3]
```diff
@@ -39,28 +39,30 @@ type Props = {
...
-        </CaptionText>
-      </CaptionWrapper>
+    // In order to be valid html, a figcaption should appear as the first or
+    // last element in a figure. We have previously made this happen
+    // (https://github.com/wellcomecollection/wellcomecollection.org/pull/9103)
+    // but we have since discovered that Firefox will read the figcaption
+    // _before_ the img alt unless the figcaption is wrapped. On balance, it
+    // makes sense to have more accessible markup at the expense of it being
+    // less valid.
+    <Wrapper width={width} style={width ? { width: `${width}px` } : undefined}>
```
[the remark this answers is not in the document]
Comment (davidpmccormick) [reply common/views/components/Caption/Caption.tsx:57]: ah no – didn't look at this. I'll have a look now

**Review by rcantin-w — APPROVED** [review]

File: common/views/components/Caption/Caption.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidpmccormick) [reply common/views/components/Caption/Caption.tsx:57]: Looks like the `style` block [didn't get taken off at the same time as the styled-component prop was added](https://github.com/wellcomecollection/wellcomecollection.org/commit/d71e5aca28d2d40affbeac83600e75b48ce6c2dc#diff-ee78530afa1e5b56f8675990f49373522856b37bb8e9b2ff974f2403717db8e7R56). I'll update this PR

**Pull request merged**